frontend/src/components/Card.tsx

```diff
@@ -99,7 +96,7 @@ export function Card({ card, onImageClick, className, editable = true }: CardPro
       </p>
 
       <div className="flex items-center gap-x-1">
-        {editable && !params.friendId ? (
+        {editable ? (
```

Comment:
Why did you remove this check? The plus/min buttons should not show when viewing a friend collection.